backend/backend.go

```diff
@@ -157,9 +156,9 @@ func Initialize(config *Config) (*Backend, error) {
 
 	// Initialize eventd
 	event, err := eventd.New(eventd.Config{
-		Store:          store,
-		Bus:            bus,
-		MonitorFactory: monitor.EtcdFactory(b.Client, "eventd"),
+		Store:           store,
+		Bus:             bus,
+		LivenessFactory: liveness.EtcdFactory(context.TODO(), b.Client),
```

Comment:
It might be a stupid question but could we pass the backend context instead, so whenever the backend stops the liveness package acts accordingly? e.g.: https://github.com/sensu/sensu-go/blob/61dc7b3a84c2d5dd4be5afe374f290535130881e/backend/liveness/liveness.go#L242